License identified in kernel/user_namespace.c:
// SPDX-License-Identifier: GPL-2.0-only

--- a/kernel/user_namespace.c.orig	2023-11-03 18:16:07.511298697 -0700
+++ b/kernel/user_namespace.c	2023-11-03 18:19:43.615291756 -0700
@@ -22,6 +22,13 @@
 #include <linux/bsearch.h>
 #include <linux/sort.h>
 
+/* sysctl */
+#ifdef CONFIG_USER_NS_UNPRIVILEGED
+int unprivileged_userns_clone = 1;
+#else
+int unprivileged_userns_clone;
+#endif
+
 static struct kmem_cache *user_ns_cachep __ro_after_init;
 static DEFINE_MUTEX(userns_state_mutex);
 
